docs/CONFIG-PROPERTIES.md

```diff
@@ -35,6 +35,7 @@
 | maintenance.mode | "enabled" or "disabled" | "none" | don't run applications etc |
 | force.fallback.counter | integer | 0 | forces fallback to other image if counter is changed |
 | newlog.allow.fastupload | boolean | false | allow faster upload gzip logfiles to controller |
+| storage.dom0.memory.minusage.percent | integer percent | 20 | min. percent of memory reserved for dom0 |
```

Comment:
Presumably the name shouldn't have "storage" in it.

Reply:
But I think the knob we need is the disable knob. Should be able to get the dom0 memory size from the build somewhere.

Reply:
That is called 
	IgnoreDiskCheckForApps GlobalSettingKey = "storage.apps.ignore.disk.check"
for storage. Thus some similar name but with memory instead of disk/storage would make sense.

Reply:
I don't think this is a correct approach -- the is a dynamic setting that controls this value precisely from the bootloader up and I see no reason not to use it @zed-rishabh 

What's the point of percentage here anyway? Why shouldn't it be an absolute value?

Reply:
>the is a dynamic setting that controls this value precisely from the bootloader up and I see no reason not to use it

@rvs Can you please share the pointer to the file where we are setting it up?